model_analyzer/config/run/run_config_generator.py

```diff
@@ -136,6 +136,42 @@ def generate_run_config_for_model_sweep(self, model, model_sweep):
                               model.triton_server_environment()))
 
     def generate_model_config_combinations(self, value):
+        configs = self._generate_model_config_combinations_helper(value)
+        if not self._default_config_in_configs(configs):
+            self._add_default_config(configs)
+        return configs
+
+    def _default_config_in_configs(self, configs):
+        """ 
+        Returns true if a default configuration exists within the
+        specified list of configs
+        """
+
+        found = False
+        for config in configs:
+            if config is None:
+                found = True
+                break
+
+            default_dynamic_batching_found = config.get(
+                "dynamic_batching") is None
+
+            instances = None
+            instance_group = config.get("instance_group")
+            if instance_group is not None:
+                instances = instance_group[0]["count"]
+            default_instances_found = instances is None or instances == 1
+
+            if default_instances_found and default_dynamic_batching_found:
+                found = True
+                break
+        return found
+
+    def _add_default_config(self, configs):
+        # Add in an empty configuration, which will apply the default values
```

Comment:
Is my understanding correct here? Empty config = default?

Reply:
I think it would be better to remove this one line method and have the logic in L156.

Reply:
This is intentional. It is self-documenting code and part of clean code.